AppController/terminate.rb

```diff
@@ -7,18 +7,19 @@
 APPSCALE_CONFIG_DIR = "/etc/appscale"
 
 module TerminateHelper
-
-
   # Erases all AppScale-related files (except database state) from the local
   # filesystem. This is used when appscale is shutdown.
   # TODO: Use FileUtils.rm_rf instead of backticks throughout this
   # method.
   def self.erase_appscale_state
+    `service appscale-controller stop`
+
     `rm -f #{APPSCALE_CONFIG_DIR}/secret.key`
     `rm -f /tmp/uploaded-apps`
     `rm -f ~/.appscale_cookies`
     `rm -f /etc/nginx/sites-enabled/appscale-*.conf`
-    `rm -f /etc/haproxy/sites-enabled/*.cfg`
+    `rm -f /etc/haproxy/app-site-enabled/*.cfg`
```

Comment:
I think this should be "app-sites-enabled". Also, I would double-check if this is still needed. Looks like there's a reference in Hermes, but the RoutingManager no longer uses intermediate files to compile the final config.

Reply:
you are right, not used anymore, I will remove it.